docs/pages/installation/windows.md

```diff
@@ -1,48 +1,126 @@
-### Windows
+---
+layout: page
+title: Windows
+permalink: installation/platforms/windows
+---
+
+## **Quick Start with Windows**
+To set up and run Meshery on Windows:
+
+1. <a href="#step1">Configure Windows and enable Docker </a>
+2. <a href="#step4">Install a Kubernetes cluster on top </a>
+3. <a href="#step5">Run Meshery</a>
+
+### **Compatibility**
+The following minimum Windows build versions are required:
+
+
+<table id="compatibility-table">
+  <tr>
+    <th id="model">Name</th>
+    <th id="model">Version</th> 
+  </tr>
+  <tr>
+    <td><a href="#wsl1">WSL1</a></td>
+    <td><b>x64</b> - Windows 7 </td>
+  </tr>
+  <tr>
+    <td><a href="#wsl2">WSL2</a></td>
+    <td><b>x64</b> - Version 1903, Build 18362; <b>ARM 64</b> - Version 2004, Build 19041</td>
+  </tr>
+  <tr>
+    <td><a href="https://docs.microsoft.com/en-us/windows/wsl/release-notes#build-18945">Custom Kernel</a></td>
+    <td>Build 18945</td>
+  </tr>
+  <tr>
+    <td><a href="https://docs.microsoft.com/en-us/windows/wsl/release-notes#build-19013">Kernel with K8s required modules</a></td>
+    <td>Build 19013</td>
+  </tr>
+</table>
+
+### **Steps**
+Perform the following steps in order:
+
+#### 1. <a name="step1" href="https://docs.microsoft.com/en-us/windows/wsl/install-win10"><b>Install Windows Subsystem for Linux (WSL)</b></a> 
+
+```bash
+dism.exe /online /enable-feature /featurename:Microsoft-Windows-Subsystem-Linux /all /norestart
+Restart-Computer
+```
 
-#### Installing the `mesheryctl` binary
+##### **Choosing your WSL version:**
 
-Download and unzip `mesheryctl` from the [Meshery releases](https://github.com/layer5io/meshery/releases/latest) page. Add `mesheryctl` to your PATH for ease of use. Then, execute:
+<h6><b><a href="https://docs.microsoft.com/en-us/windows/wsl/release-notes#build-18917" name="wsl2">WSL2</a></b> (Recommended)</h6>
+Set the default version to `WSL2`, which will be inherited by any distro you wish to use.
 
 ```bash
-./mesheryctl system start
+wsl --set-default-version 2
 ```
 
-#### Scoop
+#### 2. <b>[Install a new distro](https://docs.microsoft.com/en-us/windows/wsl/install-win10#install-your-linux-distribution-of-choice)</b>
+In this tutorial, [Ubuntu 18.04](https://www.microsoft.com/en-us/p/ubuntu-1804-lts/9n9tngvndl3q?activetab=pivot:overviewtab) will be the distro used. Feel free to use any distro of your choice.
```

Comment:
We have given the URL, then the same or partial content is again repeated in the markdown file. 
I would point to the WSL URL(Microsoft page) and omit all until Step 3 Enable Docker.

Reply:
Thanks for the review @ameyaagashe Let me explain why the headings were added in order: 

- For the command `dism.exe /online /enable-feature /featurename:Microsoft-Windows-Subsystem-Linux /all /norestart Restart-Computer`, the tutorial is written in such a way that ideally, you wouldn't need to refer to the WSL documentation. The link is there for reference.
- For the WSL2 heading, we are recommending WSL2 over WSL1, which is why the Windows documentation was updated. Won't make sense if we remove that.
- The official documentation of WSL doesn't actually point to any particular distros and as we are working with Ubuntu 18.04 here throughout the tutorial, it is worth mentioning, in my opinion. Will edit if you and other reviewers disagree though.

Reply:
@ameyaagashe Could please explain which Powershell command you are referring to?
About checking their build version, we could add a short explanation on how to do that or refer them to a freeware software like [speccy](https://www.ccleaner.com/speccy) (Link is just for reference. Not the official site).
//CC @nunix